scripts/start-local-development.sh

```diff
@@ -3,5 +3,5 @@
 set -euo pipefail
 
 ./gradlew --stop
-
-SYSTEM_CLIENT_ID=$SYSTEM_CLIENT_ID SYSTEM_CLIENT_SECRET=$SYSTEM_CLIENT_SECRET SPRING_PROFILES_ACTIVE=dev ./gradlew bootRun
\ No newline at end of file
+./gradlew clean
+SYSTEM_CLIENT_ID="bill-sclater" SYSTEM_CLIENT_SECRET="KbICZKm61'xNx+9y5Z63IvA-7(ohdu&Ubg0LF0m%\$q0suoF<P8S3E\$QV*;5h" SPRING_PROFILES_ACTIVE=dev ./gradlew bootRun
```

Comment:
Do you want to be hardcoding your secret here?

Reply:
Gone!